crates/driver/src/print_build.rs

```diff
@@ -450,6 +456,20 @@ async fn print_file(
 
     apply_manual_refs(&mut extra_kv_pairs, &graph_node.node_metadata);
 
+    fn apply_attr_string_lists(
+        extra_kv_pairs: &mut HashMap<String, Vec<String>>,
+        node_metadata: &GraphNodeMetadata,
+    ) {
+        for attr in node_metadata.attr_string_lists.iter() {
+            let t = extra_kv_pairs
```

Comment:
same comment as above (also modestly suspicious of the 4 duplicated lines of code. Rust has a great inliner I think, so having a private function you call in both cases may be nice).

Reply:
Refactored to `append_key_values` function in 525096d2c8e7dd0fc2228985b2c42cdb754d76d9